pkg/store/storeFileSystem.go

```diff
@@ -251,3 +292,69 @@ func (f *fileStore) LoadMetadata() (*fileMetadata, error) {
 	}
 	return &metadata, nil
 }
+
+// mdmExists checks if MDM managed preferences file exists and is readable
+func (f *fileStore) mdmExists() bool {
+	if f.mdmIdentifier == "" {
+		return false
+	}
+	
+	// Only check MDM on Darwin (macOS)
+	if runtime.GOOS != "darwin" {
+		return false
+	}
+	
+	mdmPath := filepath.Join("/", "Library", "Managed Preferences", f.mdmIdentifier+".plist")
+	
+	// Check if file exists and is readable
+	if _, err := os.Stat(mdmPath); os.IsNotExist(err) {
+		return false
+	}
+	
+	// Try to open file to verify readability
+	file, err := os.Open(mdmPath)
+	if err != nil {
+		return false
+	}
+	file.Close()
+	return true
+}
+
+// getMDMData reads data from MDM managed preferences plist
+func (f *fileStore) getMDMData() ([]byte, error) {
+	if f.mdmIdentifier == "" {
+		return nil, fmt.Errorf("MDM identifier not set")
+	}
+	
+	mdmPath := filepath.Join("/", "Library", "Managed Preferences", f.mdmIdentifier+".plist")
+	
+	// Read the plist file
+	// Note: In a real implementation, you'd use a proper plist parser
```

Comment:
Yeah this is interesting because I _think_ JSON is the newer OS expectation, but XML is more common.